core/vm/evm.go

```diff
@@ -29,9 +29,9 @@ import (
 	"github.com/scroll-tech/go-ethereum/params"
 )
 
-// emptyCodeHash is used by create to ensure deployment is disallowed to already
+// emptyKeccakCodeHash is used by create to ensure deployment is disallowed to already
 // deployed contract addresses (relevant after the account abstraction).
-var emptyCodeHash = codehash.EmptyCodeHash
+var emptyKeccakCodeHash = codehash.EmptyKeccakCodeHash
```

Comment:
As mentioned above, the `codehash` semantics can be use among the whole evm context and do not need to rename it as `KeccakCodeHash`

Reply:
similar here. I think `KeccakCodeHash` could reduce the confusion to others.